### ISSUE TITLE
Doc missing for building image

### ISSUE THREAD

**Issue by anotinelg** [aws/sagemaker-mxnet-inference-toolkit#123]
I am building the image for the 1.4.1 framework, from the Readme, you removed a part which is needed at least for the 1.4.1:
```
git clone https://github.com/aws/sagemaker-mxnet-serving-container.git
    cd sagemaker-mxnet-serving-container
    python setup.py sdist
```

**Comment by laurenyu** [comment]
looks like this was addressed in #124. the current Dockerfiles (and image building) have been moved to https://github.com/aws/deep-learning-containers

**Comment by anotinelg** [comment]
Thanks @laurenyu , i have some questions though:
- Do you mean that from now on, we should go to the https://github.com/aws/deep-learning-containers to create our own containers? 
- is this repository (https://github.com/aws/sagemaker-mxnet-inference-toolkit) deprecated?
- i need to build a docker image for Mxnet Elastic inference, it does not seem that the deep-learning-containers has implemented yet the EIA support for 1.6.0, the deep-learning-containers only handle 1.6.0 and CPU or GPU image. when do you plan supporting EIA for 1.6.0? 
- Meanwhile, is it valid to stick with the sagemaker-mxnet-inference-toolkit/1.4.1/py3/eia image, will it still be supported?

Thanks

**Comment by laurenyu** [comment]
> Do you mean that from now on, we should go to the https://github.com/aws/deep-learning-containers to create our own containers?

if you're using the provided Dockerfiles, yes

> is this repository (https://github.com/aws/sagemaker-mxnet-inference-toolkit) deprecated?

no. this repository is now a library released to PyPI. You can see that the Dockerfiles in https://github.com/aws/deep-learning-containers pip install this repository, e.g. https://github.com/aws/deep-learning-containers/blob/master/mxnet/inference/docker/1.6.0/py3/Dockerfile.cpu#L78

> i need to build a docker image for Mxnet Elastic inference, it does not seem that the deep-learning-containers has implemented yet the EIA support for 1.6.0, the deep-learning-containers only handle 1.6.0 and CPU or GPU image. when do you plan supporting EIA for 1.6.0?

I can't speak to the timeline, but the team that maintains the https://github.com/aws/deep-learning-containers repository have been the ones who guide that (which is why we did the repo split), so I'd recommend opening an issue in that repository.

> Meanwhile, is it valid to stick with the sagemaker-mxnet-inference-toolkit/1.4.1/py3/eia image, will it still be supported?

Our previously released ECR images will remain in ECR so that they can continue to be used, but we likely won't release any updates to them.

**Comment by anotinelg** [comment]
thanks @laurenyu  for the clarification!
